pkg/universe.dagger.io/kapp/cue.mod/pkg/github.com/renuy/kapp/source.cue

```diff
@@ -0,0 +1,138 @@
+// Kapp cli tools
+package kapp
+
+import (
+	"alpha.dagger.io/dagger/op"
+	"alpha.dagger.io/dagger"
+	"alpha.dagger.io/alpine"
+)
+
+// Install Kapp cli
+#Kapp: {
```

Comment:
Should be named `#CLI`.

To avoid `kapp.#Kapp`, instead `kapp.#CLI` is more natural.